test/unit/host_test.rb

```diff
@@ -2058,6 +2058,20 @@ def to_managed!
     end
   end # end of context "location or organizations are not enabled"
 
+  test "should not save if owner taxonomies do not match host configuration" do
+    org1 = FactoryGirl.create(:organization)
+    loc1 = FactoryGirl.create(:location)
```

Comment:
Is `create` necessary vs `.build`, or even nothing? Looks like `self.owner.organization_and_child_ids.include?(self.organization_id)` and `organization_id.present?` would resolve w/o creating the object